pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml

```diff
@@ -135,62 +145,70 @@ theme: pdf
           to learn more about supporting computer science in your state.
 
   %div.clear-both
-    %h2
-      Code.org Impact in
+    %h2.no-margin-bottom
+      Code.org's Impact in
       = state_info[:state_name_s]
-    %ul
-      %li
-        Code.org has partnered with
-        = succeed '.' do
-          = state_info[:school_district_partners_t]
-      %li
-        - if state_info[:num_affiliates_i] == 0
-          There are no Code.org affiliates in the state to offer professional development for elementary teachers.
-        - else
-          - if state_info[:num_elementary_trained_i] == 0
-            [num_affiliates_i] Code.org affiliates offers professional development for elementary teachers in [State name]
-          - else
-            = format_integer_with_commas(state_info[:num_affiliates_i])
-            Code.org
-            - if state_info[:num_affiliates_i] == 1
-              affiliate
-            - else 
-              affiliates
-            offering professional development for elementary teachers in
-            = state_info[:state_name_s]
-            have already trained
-            = format_integer_with_commas(state_info[:num_elementary_trained_i])
-            elementary
-            - if state_info[:num_elementary_trained_i] == 1 
-              teacher.
-            - else
-              teachers.
-      %li
-        Code.org has trained
-        = format_integer_with_commas(state_info[:num_ms_trained_i])
-        middle school
-        - if state_info[:num_ms_trained_i] == 1
-          teacher
-        - else
-          teachers
-        and
-        = format_integer_with_commas(state_info[:num_hs_trained_i])
-        high school
-        - if state_info[:num_hs_trained_i] == 1
-          teacher
-        - else
-          teachers
-        in
-        = state_info[:state_name_s]
-        to teach computer science. 
-    %p 
+    %p.larger
       About
       = format_integer_with_commas(state_info[:num_hoc_i])
       people in
       = state_info[:state_name_s]
-      have completed an Hour of Code! 
+      have completed an Hour of Code!
+    %ul
+      - if state_info[:school_district_partners_t] != 'no school districts in the state'
```

Comment:
It definitely is. I'd like to work with Katie on doing this in excel so that she can keep track of the logic and not prevent errors.